sql/lesson2/tutorial.md

```diff
@@ -0,0 +1,616 @@
+---
+layout: page
+title: Introduction to SQL
+---
+
+## About Lesson 2
+
+_Lesson 2_ will extend your knowledge of the **SELECT** operator by explaining some new operators, ways to sort the results and to **LIMIT** the number of results shown. You'll continue working with an online database already filled in with some data. At the end, you'll find some exercises constructing on top of the lessons 1 and 2.
+
+### Ordering query results
+
+Normally, when you write a query, you'll want your results to be shown in a specific order. Doing so is very simple knowing the operator **ORDER BY**.
```

Comment:
Suggest `Doing so is very simple knowing the operator ***ORDER BY***` should be `You can achieve this using the **ORDER BY*** operator`

Reply:
done